projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -850,6 +850,17 @@ public static function register_endpoints() {
 				'permission_callback' => __CLASS__ . '::manage_modules_permission_check',
 			)
 		);
+
+		// Get Jetpack introduction offers
+		register_rest_route(
+			'jetpack/v4',
+			'/intro-offers',
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => __CLASS__ . '::get_intro_offers',
+				'permission_callback' => __CLASS__ . '::manage_modules_permission_check',
```

Comment:
This will check for `current_user_can( 'jetpack_manage_modules' )`. Will that dis-allow this endpoint in any cases where it is needed? 
There may not be any changes needed here, but was curious.

Reply:
Good point. `view_admin_page_permission_check` is probably more appropriate here.